R/T_and_F_symbol_linter.R

```diff
@@ -1,6 +1,31 @@
 #' `T` and `F` symbol linter
 #'
-#' Avoid the symbols `T` and `F` (for `TRUE` and `FALSE`).
+#' Avoid the symbols `T` and `F`, and use `TRUE` and `FALSE` instead.
+#'
+#' @examples
+#' library(lintr)
+#'
```

Comment:
I wouldn't use library(lintr) in the examples, it shouldn't be necessary, is that recommended somewhere?

Reply:
No, not recommended anywhere. 

My thinking was that the users can basically copy-paste the examples and run it with no addition.
Otherwise, they run it, realize they forgot to `library(lintr)`.

But, if there is some downside to doing so, can easily remove these calls.

Reply:
The same's true of any package, though, so including `library(lintr)` may seem puzzling.

Users running examples _via_ `example()` will have no such issue -- `example()` will always call `library()`:

https://github.com/r-devel/r-svn/blob/e8334d998b8a3185e490f488629a7a2cc7e3825d/src/library/utils/R/example.R#L83-L84

Reply:
Okay, fair enough. 

Also, at least in RStudio, you can run the examples in the IDE itself, so they won't even need to copy-paste anything.

I have removed these calls.